app/assets/javascripts/govuk-component/option-select.js

```diff
@@ -70,6 +71,15 @@
 
   };
 
+  OptionSelect.prototype.setCheckboxAriaControlsAttributes = function setCheckboxAriaControlsAttributes(){
+    var controls = this.$optionSelect.data('input-aria-controls');
+    if (typeof controls === "string" && $('#' + controls).length > 0) {
+      this.$optionSelect.find('input[type="checkbox"]').each(function() {
```

Comment:
Do we know how long this takes to execute?

Reply:
Before
57.10ms
46.03ms
51.65ms

After
56.92ms
45.76ms
55.98ms

At CPU 6x slowdown